deploy/nginx.conf

```diff
@@ -28,6 +28,8 @@ http {
             include /etc/nginx/mime.types;
         }
 
+        client_max_body_size 100M;
```

Comment:
Maybe add a comment to keep this in sync with `settings.py`?

Reply:
Comment added.